spec/features/timeline/timeline_zooming_spec.rb

```diff
@@ -6,11 +6,11 @@
 describe "Timeline zooming", reset: false do
```

Comment:
This needs eight tests:

 * when an off-center temporal constraint is selected, clicking the zoom in button centers on the selected temporal constraint
 * when an off-center temporal constraint is selected, clicking the zoom out button centers on the selected temporal constraint
 * when an off-center temporal constraint is selected, zooming in with the mouse wheel centers on the mouse cursor
 * when an off-center temporal constraint is selected, zooming out with the mouse wheel centers on the mouse cursor
 * when an off-center date is focused, clicking the zoom in button centers on the selected temporal constraint
 * when an off-center date is focused, clicking the zoom out button centers on the selected temporal constraint
 * when an off-center date is focused, zooming in with the mouse wheel centers on the mouse cursor
 * when an off-center date is focused, zooming out with the mouse wheel centers on the mouse cursor